sources/editor/Stride.Core.Assets.Editor/View/AssetViewUserControl.xaml.cs

```diff
@@ -310,11 +310,11 @@ private static void CanBeginEditCommand(object sender, CanExecuteRoutedEventArgs
             e.CanExecute = control.CanBeginEdit();
         }
 
-        private static void CanBeginEditEvent(object sender, CancelRoutedEventArgs e)
-        {
-            var control = (AssetViewUserControl)sender;
-            e.Cancel = !control.CanBeginEdit();
-        }
+        //private static void CanBeginEditEvent(object sender, CancelRoutedEventArgs e)
```

Comment:
Better remove code instead of commenting out.

Reply:
> 
> 
> I know we did use AvalonDock in the past, but we had some issues. Maybe the situation is reversed now.
> 
> Does switching and saving the editor layout still work with AvalonDock? I seem to remember the XML format was different.

Not quite sure what you mean in the past? You mean this specific fork's?
As far as I can tell, we've been using Xceed's AvalonDock (even in the official v3.1), and this fork is still based off Xceed's AvalonDock, but prior to their license change (as far as I can tell).

I've tested it running the editor, docking a panel somewhere else then reloading, and it seems to remember it properly.
Nothing crashing as far as I can tell... Probably needs more eyes to confirm it's working, though.